helm-charts/powerdns/pdns-data-manager/dnsdata.yaml

```diff
@@ -0,0 +1,81 @@
+zones:
```

Comment:
This data currently appears to be static, and would need to be "manually" fixed to change all of the IP address entries to the appropriate IP address...

Is the intent that this file will be dynamically templated by the deployment, similarly to what you were doing with the lifecycle post hook script? Or is this something that the user is expected to manually fix up?

Reply:
It was expected for the user to update the input yaml file. However, since we do not need this utility anymore, I will be removing this from the commit.